extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -97,86 +97,79 @@ const renderSystemLayerElement = (element, children) => (
     <div class="i-amphtml-story-has-new-page-notification-container">
       <div class="i-amphtml-story-has-new-page-text-wrapper">
         <span class="i-amphtml-story-has-new-page-circle-icon" />
-        <div class="i-amphtml-story-has-new-page-text">
-          {localize(
-            element,
-            LocalizedStringId_Enum.AMP_STORY_HAS_NEW_PAGE_TEXT
-          )}
-        </div>
+        <div class="i-amphtml-story-has-new-page-text"></div>
```

Comment:
We should keep the localization key here, spreading them out between the template and a post processing method sounds wrong.

You could do something like `*-18n-inner-text`